docs/doc/code-review.md

```diff
@@ -25,11 +32,10 @@ PRs should have proper name (as per task name), should contain link to working d
 ### Code formatting
 
 - `console.log` statements should not be left in final version of the code, unless it's part of the functionality
-- Please advise your peer to fix irregular indentations and remove redundant empty lines.
-- Please advise your peer to put newline at the end of every file.
-- Unnecessary comments should be avoided. Advise your peer to find a way to express their intent through expressive variable names or by abstracting part of the code into properly named function.
-- Magic numbers are [bad](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)
-- Add an empty lines at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)
+- Fix irregular indentations and remove redundant empty lines.
+- Put an empty line at the end of every file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1). Tune your code editor's settings so it does this for you.
+- Unnecessary comments should be avoided. Find a way to express the intent through expressive variable names or by abstracting part of the code into properly named function. Comments, if there are any, should explain "why", not "what".
```

Comment:
`express` through `expressive`. Maybe `express the intent through descriptive variable names` is better?

Reply:
`into a properly named function` or `into properly named functions`

Reply:
I like `descriptive` more as it more specific than `proper`.